test/integration.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
not sure what docker-outside-of-docker means.  Our established plan is to use docker, and if someone else wants to not use it then they're on their own.